backend/cms/views/general/general.py

```diff
@@ -1,34 +1,29 @@
-from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
 
 
-@login_required
-def index(request):
-    return render(request, 'dashboard.html')
-
 def handler400(request):
     ctx = {'code': 400, 'title': 'Bad request', 'message': 'Die Anfrage war fehlerhaft aufgebaut.'}
```

Comment:
I think these should be english by default. Maybe we can support i18n.

Reply:
I will add i18n in a future pull request and I will also translate all hardcoded strings into English.